### PR TITLE
custom_params: add subscript and superscript params

### PR DESCRIPTION
**Pull request by m4rc1e**
I've noticed my VFs don't contain the subscript and superscript OS/2 attribs. Since we're pushing hard on OpenSans VF, I feel we need to add them.

To start with, I've only added "openTypeOS2SubscriptYOffset". I've got it to work but I don't know if my implementation is correct. If someone can review and confirm my current approach is ok, I'll add the rest of the attribs and make sure they're ordered better.

### CODE REVIEW

**Comment by m4rc1e** [comment]
Alright, I'm now happy with this.

I'll start reading up on tox as well.

File: tests/builder/custom_params_test.py [hunk 1]
```diff
@@ -120,31 +123,37 @@ def test_set_fsSelection_flags(self):
...
-        self.set_custom_params()
-        self.assertEqual(self.ufo.info.postscriptUnderlineThickness, 100)
+    def test_integer_parameters(self):
+        """Test casting glyphsapp customParameters whose values are just
+        integers into ufo equivalents."""
+        integer_params = [
+            "underlinePosition",
+            "underlineThickness",
+            "underlinePosition",
+            "underlineThickness",
```
Comment (anthrotype) [inline tests/builder/custom_params_test.py]: You're testing twice the same parameters.
Also the strikeoutSize and Position are missing here

Comment (m4rc1e) [reply tests/builder/custom_params_test.py]: Good catch. Thank you.

**Comment by m4rc1e** [comment]
ping @anthrotype

**Comment by anthrotype** [comment]
To be honest, I don't like this feature too much. Setting advance width to 0 is clear and easy (we do that already), whereas also shifting the mark glyph (as well as all its composites, anchors, etc.) to the left seems a bit arbitrary to me. It would also change all the glyf and GPOS tables if one rebuilds the same source font after glyphsLib would introduce this feature (which is actually what you want, in order to reduce the diff noise in your comparison tool).
But if you or others feel strongly about the need for implementing this, I am not going to veto it.
I personally would prefer if designers explicitly drew their combining mark glyphs with a zero advance width to begin with, instead of relying on the compiler to modify that at export time.

**Comment by m4rc1e** [comment]
Think you meant #547

**Comment by anthrotype** [comment]
oops, yes :)

**Comment by anthrotype** [comment]
@m4rc1e note that if you want these custom params to be set in the generated VF, you will have to set them on the _masters_, not just on the instances. If they vary across masters, an MVAR table will contain deltas for them.
If you only set them on the instances, they won't be applied for the VF, but only for the static instances.

**Comment by m4rc1e** [comment]
I'm aware sir and thank you for cutting a new release!

**Pull request merged**